### PR TITLE
refactor(locksmith): Remove need to refresh token

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/hooks/useProvider.ts [hunk 1]
```diff
@@ -35,18 +35,17 @@ export const useProvider = (config: any) => {
...
+    if (account) {
       setStorage('account', account)
     }
-
-    if (!getStorage('network') && network) {
+    if (network) {
       setStorage('network', network)
     }
-  }, [account, network])
+  }, [account, network, setStorage])
```
Comment (searchableguy) [inline unlock-app/src/hooks/useProvider.ts:48]: This was a bug where local storage account won't be updated when you switch directly using your wallet.

**Comment by kalidiagne** [comment]
@searchableguy  i think we also need to change something inside `openapi`, for example we have `refresh-token` there

**Comment by searchableguy** [comment]
Yup. Let me remove the that part. Thanks.

**Review by julien51 — APPROVED** [review]
LG

**Pull request merged**